pyproject.toml

```diff
@@ -2,7 +2,7 @@
 authors = ["Joao Da Silva <joao.silva1@avast.com>", "Michal Mrazek <michal.mrazek@avast.com>"]
 description = "CLI tool for managing ML projects on Vertex AI"
 name = "wanna-ml"
-version = "0.0.0"
+version = "0.2.4"
```

Comment:
Also I would keep this as 0.0.0 so its obvious that it is a placeholder. This value is updated by CI with git tag.